news/2017-09-21-beatmap-spotlights-july-august-2017.md

```diff
@@ -0,0 +1,259 @@
+
+This map's special patterning is what makes it stand out. Everything is perfectly aligned to the grid's axes and combined with expressive jump usage and unusual rhythms here and there, unique visuals and gameplay are created.
+
+The same style is used for every difficulty, which makes the mapset feel very coherent and professional. Players who are interested in playing something different from the usual mapping meta should be checking this one out right now!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/dreadnought.jpg)](https://osu.ppy.sh/s/361306)
+
+[Cherry Blossom](https://osu.ppy.sh/u/1156742) has yet again managed to create a map which is not only insanely clean but also very fun to play in general, with an absolutely clever use of symmetry and a very fitting use of spacing for the streams, underlining properly the intensity of the various instruments. If you like challenging stream maps, you can't go wrong with this one!
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/true-blue.jpg)](https://osu.ppy.sh/s/643803)
+
+Wait, what year it is? Eurobeat in 2017?! This time we have a remix of the song True Blue mapped by [iloveyou4ever](https://osu.ppy.sh/u/4964596), he previously mapped the original song and got in the spotlights too, but let's talk about this beautiful map (and song). We will see a good follow of the vocals and instrumental as well some piano, even if the map is 164 bpm you will enjoy it with a lot of streams and some high slider velocity.
+
+The song and the map itself it feels relaxing to play even with the amount of notes it has, as it contains a good structure and a consistent mapping, in my opinion, it's the kind of map that even if you manage to get full combo, you are gonna still replaying it to get a good score on it, and also, don't forget to give it a chance with HardRock and Hidden!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/reminiscing.jpg)](https://osu.ppy.sh/s/643269)
+
+Featured as a guest difficulty for [toybot](https://osu.ppy.sh/u/2848604)'s set, we have Reminiscing by [tasuke912](https://osu.ppy.sh/u/2774767)! An approval sized map using complex yet intuitive 1/6 and 1/3 snapping combination throughout the map, making it quite challenging for a 5.94 Star Rated beatmap.
+
+Rhythms used are fairly straightforward as we can expect from a drum and bass track, but the relatively high BPM will keep your hands hot and busy! This beatmap is definitely a must have for every high tier taiko player.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/false-cross.jpg)](https://osu.ppy.sh/s/615067)
+Despite a fairly repetitive song, [butter0414](https://osu.ppy.sh/u/7935022) managed to create a set with a good variety of patterning to dodge the song’s natural tedium, with the patterning increasingly gradually to encompass quintuplets mixed with ⅙ streams.
```

Comment:
patterning is used too much in a single sentence. Switch the second for density instead?